### ISSUE TITLE
Action doesn't recognize Oracle as a valid language

### ISSUE THREAD

**Issue by InfernapeXavier** [joshcai/leetcode-sync#6]
```
Language oraclesql does not have a registered extension.
```

The mapping for Oracle SQL submissions is missing.
Should be an easy fix, just need to change the mapping to:

```
const LANG_TO_EXTENSION = {
  'bash': 'sh',
  'c': 'c',
  'cpp': 'cpp',
  'csharp': 'cs',
  'golang': 'go',
  'java': 'java',
  'javascript': 'js',
  'kotlin': 'kt',
  'mysql': 'sql',
  'oraclesql': 'sql', //Added this
  'php': 'php',
  'python': 'py',
  'python3': 'py',
  'ruby': 'rb',
  'rust': 'rs',
  'scala': 'scala',
  'swift': 'swift',
};
```

_Suggestion_ : Logging the error and then _skipping_ the file may be a better way to handle it, instead of failing the Action at this point.

**Comment by joshcai** [comment]
Thank you for the bug report, @InfernapeXavier! I have created a new release 'v1.3' that should include your suggested fix. Let me know if that doesn't work for you, and I can take another look.

Also, thanks for your suggestion. For some context on why the action explicitly fails - the action was designed to be retriable after any error and to cover all submissions. If we skipped, then even after you re-run the syncer with the new code, the skipped submission would still not be present if a newer submission had already been synced. This way, the user doesn't have to re-create the repo and are also aware of any issues that are present.
